app/routes/application.js

```diff
@@ -0,0 +1,17 @@
+import Route from '@ember/routing/route';
+import { inject } from '@ember/service';
+export default Route.extend({
+  session: inject(),
+  beforeModel: function() {
+    return this.get('session').fetch().catch(function() {});
+  },
+  actions: {
+    signIn: function() {
+      this.get('session').open('firebase', { provider: 'google'}).then(function() { // function(data)
```

Comment:
If you are not using result from the promise - just remove it `.then(function(){ // function(data) }`